docs/console-virtual-terminal-sequences.md

```diff
@@ -101,29 +101,30 @@ Cursor movement will be bounded by the current viewport into the buffer. Scrolli
 The following commands control the visibility of the cursor and its blinking state. The DECTCEM sequences are generally equivalent to calling [**SetConsoleCursorInfo**](setconsolecursorinfo.md) console API to toggle cursor visibility.
 
 
-| Sequence      | Code    | Description                  | Behavior                  |
+| Sequence | Code | Description | Behavior |
 |---------------|---------|------------------------------|---------------------------|
-| ESC \[ ? 12 h | ATT160  | Text Cursor Enable Blinking  | Start the cursor blinking |
-| ESC \[ ? 12 l | ATT160  | Text Cursor Disable Blinking  | Stop blinking the cursor  |
-| ESC \[ ? 25 h | DECTCEM | Text Cursor Enable Mode Show | Show the cursor           |
-| ESC \[ ? 25 l | DECTCEM | Text Cursor Enable Mode Hide | Hide the cursor           |
-
+| ESC \[ ? 12 h | ATT160 | Text Cursor Enable Blinking | Start the cursor blinking |
+| ESC \[ ? 12 l | ATT160 | Text Cursor Disable Blinking | Stop blinking the cursor |
+| ESC \[ ? 25 h | DECTCEM | Text Cursor Enable Mode Show | Show the cursor |
+| ESC \[ ? 25 l | DECTCEM | Text Cursor Enable Mode Hide | Hide the cursor |
 
+> [!TIP]
+> The enable sequences end in a lowercase H character (`h`) and the disable sequences end in a lowercase L character (`l`).
```

Comment:
You could even joke about how they look like switch toggles/GPIO lines, [H]igh and [L]ow.

Reply:
I, uh.... I'm not sure about that.